packages/tpetra/core/test/MultiVector/MultiVector_MicroBenchmark.cpp

```diff
@@ -128,10 +131,9 @@ namespace { // (anonymous)
             // This is the special case code that actually executes inside KokkosKernels for alpha=1
             X_lcl(i,0) += ONE * Y_lcl(i,0);
```

Comment:
Consider the following changes:
  1. Use LocalOrdinal (32 bits) rather than `size_t` (64 bits) as the loop counter
  2. Put the value of ONE inside the kernel as a constexpr, to minimize variable capture
  3. Use 1-D Views.

The point of this kernel is to establish a baseline by being as lightweight as possible.